src/tests/Microsoft.Diagnostics.Monitoring.EventPipe/EventLogsPipelineUnitTests.cs

```diff
@@ -181,8 +181,10 @@ public async Task TestLogsUseAppFiltersAndFilterSpecs()
         /// <summary>
         /// Test that log events are collected for wildcard categories.
         /// </summary>
-        [Fact]
-        public async Task TestLogsWildcardCategory()
+        //https://github.com/dotnet/diagnostics/issues/2568
+        //[Fact]
+        //public async Task TestLogsWildcardCategory()
+        private async Task TestLogsWildcardCategory()
```

Comment:
Please only disable when running on Windows (for both tests). All test runs in the last 7 days with failures of these two tests have only failed on Windows: https://runfo.azurewebsites.net/search/tests/?q=started%3A%7E7+definition%3Adiagnostics-public-ci

Reply:
I've seen failures (at least locally) on Linux too.

Reply:
Though, I do need to double check that these Linux failures were one of these tests.

Reply:
I've made the change to only skip on Windows.